packages/@react-aria/i18n/src/index.ts

```diff
@@ -11,14 +11,13 @@
  */
 
 export {I18nProvider, useLocale} from './context';
-export {useMessageFormatter} from './useMessageFormatter';
 export {useLocalizedStringFormatter} from './useLocalizedStringFormatter';
 export {useDateFormatter} from './useDateFormatter';
 export {useNumberFormatter} from './useNumberFormatter';
 export {useCollator} from './useCollator';
 export {useFilter} from './useFilter';
 
-export type {FormatMessage} from './useMessageFormatter';
+export type {useMessageFormatter, FormatMessage} from './useMessageFormatter';
```

Comment:
By only exporting the type for `useMessageFormatter` here, it stays in the generated TS definitions, but is excluded from the Parcel-built JS.